build/build.ps1

```diff
@@ -9,7 +9,7 @@ Param(
     [ValidateSet("All","x64","x86")][string]$Architecture = "All",
     [string]$HomePath = "$env:NR_DEV_HOMEROOT",
     [string]$gpgKeyPath = "",
-    [switch]$SkipProfilerBuild = $false,
+    # [switch]$SkipProfilerBuild = $false,
```

Comment:
Can we delete this if it's no longer needed?

Reply:
I left this and the related block below commented out in case we need to come back someday.

Reply:
I think deleting it is fine. We can use source control to bring it back.